passy_reader.c

```diff
@@ -410,6 +410,74 @@ NfcCommand passy_reader_state_machine(Passy* passy, PassyReader* passy_reader) {
                 body_offset += Le;
             } while(body_offset < body_size);
             passy_log_bitbuffer(TAG, "DG1", passy_reader->DG1);
+        } else if(passy->read_type == PassyReadDG7) {
```

Comment:
Can you move this down?  it is easier to find things if the DG files are numeric and if DG2 is before DG7